webapp/templates/caveat.html

```diff
@@ -1,3 +1,3 @@
 <section class="caveat">
-  The web-platform-tests dashboard is a work-in-progress. The reported results do not necessarily reflect the true capabilities of each web browser, so they should not be used evaluate or compare feature support. Each browser listed here is currently tested at its latest stable release.
+  wpt.fyi is a work in progress. The reported results do not necessarily reflect the true capabilities of each web browser, so they should not be used evaluate or compare feature support.
```

Comment:
Or remove `a`, i.e. `wpt.fyi is work-in-progress`, which avoids the technicality of countable `work`.

No just kidding. This LGTM.

Reply:
In "wpt.fyi is work-in-progress", the hyphenated phrase is a predicative adjective instead of attributive, unless such hyphenated form isn't usually used predicatively.

Reply:
Ah, without "a". Yes, that would be grammatically correct I think. Didn't get that's what you meant initially.

Enough grammar for today :)